implants/eldritch/src/file.rs

```diff
@@ -48,58 +58,73 @@ impl<'v> UnpackValue<'v> for FileLibrary {
 // This is where all of the "file.X" impl methods are bound
 #[starlark_module]
 fn methods(builder: &mut MethodsBuilder) {
-    fn append(_this: FileLibrary, path: String, content: String) -> NoneType {
```

Comment:
Starlark errors if not _this.
Tried using `#[allow(dead_code)]` above the function definition but the error persisted.

`Receiver parameter must be named `this` or have `#[starlark(this)]` annotation`
![image](https://user-images.githubusercontent.com/7121375/200078550-59ca9418-ccba-473e-bd17-5592d58e29e9.png)

Reply:
Doing this:
``` Rust
    #[starlark(this)]
    fn append(_this: FileLibrary, path: String, content: String) -> anyhow::Result<NoneType> {
```

Throws the error.
![image](https://user-images.githubusercontent.com/7121375/200078957-0c08fcd3-5f4b-4a00-9d99-0d32276db3a3.png)